core/src/main/java/psiprobe/beans/ContainerWrapperBean.java

```diff
@@ -54,6 +68,9 @@
   @Inject
   private Map<String, ResourceResolver> resourceResolvers;
 
+  @Autowired
```

Comment:
@Aaron-LS Please make this @Inject - spring prefers this when on the path for this type of injection.